operations/agent-flow-mixin/dashboards/prometheus.remote_write.libsonnet

```diff
@@ -347,6 +347,70 @@ local stackedPanelMixin = {
         ])
       ),
 
+      // Scrape success rate
```

Comment:
The reason I called this dashboard `prometheus.remote_write` was to have component-specific dashboards; this came from the understanding that not all users will use the same set of components in a pipeline, so combining metrics from two components in a single dashboard wouldn't necessarily be useful for all users.

In this particular case, not everyone using a `prometheus.remote_write` component also has a `prometheus.scrape` component in the same pipeline (i.e., users using OpenTelemetry components but exporting metrics to Prometheus). 

Can we make a new dashboard for `prometheus.scrape`?

Reply:
I would prefer we instead rename this dashboard to "Prometheus Metrics" and I could make separate collapsible rows for different components. 

The reason is that we have a lot of components and it'd be quite frustrating to have to jump between tabs to correlate the issues in the pipeline that happen in different components. Poor user experience overall. When assessing system health, a high level dashboard is a better starting point for identifying where the issue is and I believe we should provide this in the first instance. 

For the OTEL-based pipelines, we could also have a dashboard with rows for different components or telemetry signals.

Reply:
Can this be discussed with the rest of the maintainers offline so we can get consensus on an approach for how to handle and expand these dashboards?

Reply:
I've addressed the feedback as discussed.

Reply:
For public record, the offline consensus was to create dashboards aligned with component namespaces, with each row being for a different component, and the order of the rows roughly reflecting the order on the write path for that namespace.